Source/panels/spell_book.cpp

```diff
@@ -81,9 +81,9 @@ spell_type GetSBookTrans(spell_id ii, bool townok)
 
 void InitSpellBook()
 {
-	pSpellBkCel = LoadCelAsCl2("Data\\SpellBk.CEL", static_cast<uint16_t>(SidePanelSize.width));
-	pSBkBtnCel = LoadCelAsCl2("Data\\SpellBkB.CEL", gbIsHellfire ? 61 : 76);
-	pSBkIconCels = LoadCelAsCl2("Data\\SpellI2.CEL", 37);
+	pSpellBkCel = LoadCelAsClx("Data\\SpellBk.CEL", static_cast<uint16_t>(SidePanelSize.width));
+	pSBkBtnCel = LoadCelAsClx("Data\\SpellBkB.CEL", gbIsHellfire ? 61 : 76);
+	pSBkIconCels = LoadCelAsClx("Data\\SpellI2.CEL", 37);

```

Comment:
Nit: At this point it would be implied that the graphics are being loaded as clx.

Reply:
Shall I remove all the `AsClx` suffixes throughout?

Reply:
if you agree that it would be cleaner :)

Reply:
Done!